src/openrct2/scenario/ScenarioRepository.cpp

```diff
@@ -513,8 +513,7 @@ class ScenarioRepository final : public IScenarioRepository
         // If the user has a Steam installation.
         if (!File::Exists(mpdatPath))
         {
-            mpdatPath = Path::ResolveCasing(
-                Path::Combine(_env->GetDirectoryPath(DIRBASE::RCT1), u8"RCTdeluxe_install", u8"Data", u8"mp.dat"));
+            mpdatPath = _env->FindFile(DIRBASE::RCT1, DIRID::DATA, "mp.dat");
```

Comment:
This is superfluous, when it gets here it has already checked for the existence of mp.dat in the usual location.

Reply:
So should I just remove the `PATHID::MP_DAT` entry, and replace the top line with this line?

Reply:
Fixed